packages/atomic/src/components/search/atomic-did-you-mean/atomic-did-you-mean.new.stories.tsx

```diff
@@ -56,8 +56,6 @@ const searchPlay: (
   )?.find((el) => el.getAttribute('part') === 'submit-button');
 
   await step(`Search "${query}"`, async () => {
-    searchBox!.value = '';
-
```

Comment:
I am pretty sure this is what was causing the flakiness. I put this here because storybook was keeping the old value when switching stories on storybook. I tried to fix it but it only seem to cause more problems. :/